settings/config.go

```diff
@@ -154,3 +155,38 @@ func WriteConfigToFile(file string) error {
 
 	return err
 }
+
+func EditConfigFileParameter(file string, parameter string) error {
```

Comment:
We use viper to manage the config files, please don't manually edit the settings file.